src/integrations/express.ts

```diff
@@ -156,6 +157,36 @@ function getExpressRouter({ api }: GetExpressRouterContext) {
       }
     }
   )
+
+  if (api.enableEventStream()) {
+    router.get(
```

Comment:
a private method "registerChangeStreamRoute" that calls the `rounter.get` method from your implementation would be beneficial refactoring.